src/main/java/org/zalando/nakadi/service/AdminService.java

```diff
@@ -59,7 +60,8 @@ public AdminService(final AuthorizationDbRepository authorizationDbRepository,
 
     public List<Permission> getAdmins() {
         try {
-            return addDefaultAdmin(resourceCache.get(ADMIN_RESOURCE, authorizationDbRepository::listAdmins));
+            return ImmutableList.copyOf(
```

Comment:
This code is in a hot path. Do you think it would be possible to avoid these copies?